OKTA-807236 - Remove rate limit content (#5101)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/reference/rl-global-enduser/index.md

| `/login/sso_iwa_auth`<br>Eligible for workforce multiplier                                                                                                  | 100              | 600              | 600     | 600        | 500                   |
| `/api/{apiVersion}/radius`<br>Eligible for workforce multiplier                                                                                             | 100              | 600              | 600     | 600        | 600                   |
| `/login/token/redirect`<br>Eligible for dynamic scale and workforce multiplier                                                                                                             | 100              | 600              | 600     | 600        | 600                   |

> | <ApiLifecycle access="ie" /> **Identity Engine APIs:**</br> Identity Engine rate limits are configured to support 1000 Identity Engine authentication flows per minute. That is, depending on authentication flows, some endpoint limits may differ.                                                                                                        |

Reviewer: ```suggestion
| <ApiLifecycle access="ie" /> **Identity Engine APIs:**</br> Identity Engine rate limits are configured to support 1000 Identity Engine authentication flows per minute. Depending on the authentication flow, some endpoint limits may differ.                                                                                                        |
```
